packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -126,15 +127,18 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
                 </ConfirmationsHeading>
                 <StyledCheckbox isChecked={connectedConfirmed} onClick={toggleConnectConfirmation}>
                     <Translation id="TR_DEVICE_CONNECTED_CONFIRMATION" />
+                    <TooltipIcon message="TR_DEVICE_CONNECTED_TOOLTIP" />
                 </StyledCheckbox>
                 <StyledCheckbox isChecked={termsConfirmed} onClick={toggleTermsConfirmation}>
                     <Translation
                         id="TR_TERMS_AND_PRIVACY_CONFIRMATION"
                         values={{
-                            privacy: chunks => (
-                                <StyledLink href="https://trezor.io">{chunks}</StyledLink>
-                            ), // TODO: replace URL
-                            terms: chunks => (
+                            coordinator: chunks => (
+                                <StyledLink href="https://github.com/zkSNACKs/WalletWasabi/tree/master/WalletWasabi/Legal/Assets">
```

Comment:
how about putting the url in `@trezor/urls` package so it's covered by tests trying if external links are alive?

Reply:
738c1974eba8f603d2898768d895e94a7aa19c27